<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserItemWidget</class>
 <widget class="QWidget" name="UserItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成员属性</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	color:rgb(57,129,252);
	background: rgb(255,255,255);
	border: 1px solid rgb(57,129,252);
	border-radius: 2px;
	min-height: 28px;
	min-width: 76px;
}
QPushButton:hover, QPushButton:pressed {
	background:rgb(57,129,252);
	color: rgb(255,255,255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="lblUsername"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnUserAttrs">
     <property name="text">
      <string>成员属性</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
